build.sbt

```diff
@@ -1,13 +1,14 @@
 // https://typelevel.org/sbt-typelevel/faq.html#what-is-a-base-version-anyway
 ThisBuild / tlBaseVersion := "0.0" // your current series x.y
 
-ThisBuild / organization := "io.pig"
-ThisBuild / organizationName := "Pig.io"
+ThisBuild / organization := "pink.cozydev"
+ThisBuild / organizationName := "Cozydev.pink"
```

Comment:
Alternatively we could go with just `Cozydev`? Is the `pink` part of our name?

Reply:
Update: we went with `CozyDev`